app/client/cli/debug.go

```diff
@@ -25,8 +25,14 @@ const (
 	PromptTogglePacemakerMode    string = "TogglePacemakerMode"
 	PromptShowLatestBlockInStore string = "ShowLatestBlockInStore"
 
-	defaultConfigPath  = "build/config/config1.json"
+	defaultConfigPath = "build/config/config1.json"
```

Comment:
I've changed this in LocalNet PR - https://github.com/pokt-network/pocket/blob/1261079d7bfe62b747809ad52470c3720888423f/app/client/cli/debug.go#L41 - and we could make use of environment variables instead of passing arguments. I can make adjustments/fix conflicts after the other PR is merged. We can keep both - envars and arguments, in that case I can make changes in LocalNet PR after this is merged.